oops_program_1.cpp

```diff
@@ -0,0 +1,58 @@
+#include<iostream>
+Class student
+{
+   Protected:
+       int roll_number;
+   public:
+       void get_number(int);
+       void put_number(void);
+ }
+Void student::get_number(int a)
```

Comment:
Please use appropriate parameter names. parameter name 'a' does not signify anything. 
One of the key changes from College Projects to Enterprise software development is naming conventions and style including formatting and indentation. Also important is code comments.